front/app/containers/Admin/projects/all/index.tsx

```diff
@@ -112,52 +120,91 @@ const AdminProjectsList = memo(({ className }: Props) => {
     search,
   });
 
-  const { data: publishedAdminPublications } = useAdminPublications({
-    publicationStatusFilter: ['published'],
+  const { data: statusCounts } = useAdminPublicationsStatusCounts({
```

Comment:
Using this for the counts in the tabs if we're not loading lists of admin publications anymore for non-active tabs.